### PR TITLE
fix(README) Better build instructions

### PR DESCRIPTION
**Pull request by xiaoxiao921**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,20 +54,37 @@ cd YimMenu
...
+  cmake ..
+  ```
+
+  Now, you will be able to open the solution, and build it.
 
 
 ## Staying Up To Date
 
-```bash
-git pull https://github.com/YimMenu/YimMenu.git
```
Comment (tupoy-ya) [inline README.md:67]: But CMake doesn't pull YimMenu changes for you.

File: README.md [hunk 2]
```diff
@@ -54,20 +54,37 @@ cd YimMenu
...
-```bash
-mkdir build && cd build
-cmake ..
-```
-Now, you will be able to open the solution, and simply build it in Visual Studio.
+- On Windows
+
+  - Visual Studio
+
+    If you only use Visual Studio and don't want to mess with command lines, Visual Studio have a CMake extension that does all the work.
```
Comment (Yimura) [inline README.md]: `has a CMake extensions`

File: README.md [hunk 3]
```diff
@@ -54,20 +54,37 @@ cd YimMenu
...
+
+    Make sure it is [installed](https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170#installation).
+
+    Then, just open Visual Studio, Open a local folder, and select the `YimMenu` folder that the `git clone` command just created.
```
Comment (Yimura) [inline README.md]: `Open` with random capitalised letter after a comma.

File: README.md (hunk 1 above)
Comment (Yimura) [reply README.md:67]: `git pull <url>` isn't a valid command either.
Just use `git pull`

Though it should still be included for those fetching from remote.

**Review by Yimura — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tupoy-ya) [reply README.md:67]: This is an instruction to stay up to date which people would probably use when they make a fork, so having remote in `git pull` should help them to merge.

**Comment by tupoy-ya** [comment]
Is
```bash
git clone https://github.com/YimMenu/YimMenu.git || echo "You don't have git installed, install it from https://git-scm.com/download/win"
```
Valid in PS?

**Comment by xiaoxiao921** [comment]
> Is
> 
> ```shell
> git clone https://github.com/YimMenu/YimMenu.git || echo "You don't have git installed, install it from https://git-scm.com/download/win"
> ```
> 
> Valid in PS?

PS v7 and above, version shipped by default is v5

File: README.md [hunk 4]
```diff
@@ -42,33 +42,61 @@ CMake is used to generate our project files, if you haven't used it before we wi
...
-```bash
-git clone https://github.com/YimMenu/YimMenu.git || echo "You don't have git installed, install it from https://git-scm.com/download/win"
-```
+- Make sure that you have installed Git and CMake.
 
-Go into the directory you just cloned:
-```bash
-cd YimMenu
-```
+- Clone the repository including submodules:
```
Comment (tupoy-ya) [inline README.md]: We don't use submodules anymore

File: README.md (hunk 1 above)
Comment (Yimura) [reply README.md:67]: The keeping up-to-date section was for people who were just git cloning YimMenu locally and building (was also written before we built DLL's automatically and published them as a release).
It was never written in a way for forks to stay up-to-date.

**Review by Yimura — APPROVED** [review]

**Pull request merged**